RecoParticleFlow/PFProducer/plugins/kdtrees/KDTreeLinkerTrackHcal.cc

```diff
@@ -13,8 +13,8 @@ DEFINE_EDM_PLUGIN(KDTreeLinkerFactory,
 KDTreeLinkerTrackHcal::KDTreeLinkerTrackHcal()
   : KDTreeLinkerBase()
 {
-  setCristalPhiEtaMaxSize(0.2);
-  setPhiOffset(0.32);
+  cristalPhiEtaMaxSize_ = 0.2;
+  phiOffset_ = 0.32;
```

Comment:
these should be moved to the member initializer list before the constructor body

Reply:
That did not work: I think it's because these are members of the base class and not this one. These values could be maybe passed to the constructor of the base class, but then the base constructor would have to be changed, as well as the signature of the corresponding factory and other code. I don't think all of that should be changed in a cleanup PR.

Reply:
ok, thanks for checking.